bundle/run.sh

```diff
@@ -512,11 +512,11 @@ done
 ## configuration
 cat /configs/resource-aggregate.yaml | yq e "\
   .apis.grpc.address = \"${RESOURCE_AGGREGATE_ADDRESS}\" |
-  .apis.grpc.authorization.http.tls.useSystemCAPool = true |
+  .apis.grpc.authorization.http.tls.useSystemCaPool = true |
```

Comment:
Ca are two words. It's Certificate Authority, so it cannot be Ca.

Reply:
We will keep `useSystemCAPool`.